apps/src/templates/courseOverview/CourseOverviewTopRow.js

```diff
@@ -52,14 +35,11 @@ export default class CourseOverviewTopRow extends Component {
             color={Button.ButtonColor.blue}
           />
         ))}
-        {experiments.isEnabled(experiments.ASSIGNMENT_UPDATES) && (
-          <SectionAssigner
-            sections={sectionsForDropdown}
-            showAssignButton={showAssignButton}
-            courseId={id}
-            assignmentName={title}
-          />
-        )}
+        <SectionAssigner
+          sections={sectionsForDropdown}
+          showAssignButton={showAssignButton}
+          courseId={id}
+        />
```

Comment:
Is assignmentName gone here on purpose?

Reply:
Yes. `assignmentName` is only used in the confirm hidden assignment dialog and since courses can't be hidden the user will never see that dialog when assigning a course from the CourseOverviewTopRow.